modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseQueryWaitStrategy.java

```diff
@@ -0,0 +1,17 @@
+package org.testcontainers.couchbase;
+
+import com.couchbase.client.core.service.ServiceType;
+import com.couchbase.client.java.Bucket;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @author tchlyah
+ * Created on 12/02/2020
+ */
+@Slf4j
+public class CouchbaseQueryWaitStrategy extends CouchbaseServiceWaitStrategy {
```

Comment:
should be `CouchbaseQueryServiceWaitStrategy`

Reply:
Ah sorry, fixing it